src/components/course/data/utils.jsx

```diff
@@ -167,9 +164,29 @@ export const findEnterpriseOfferForCourse = ({
         return false;
       }
     }
-
     return true;
   });
+
+  // use offer that has no bookings limit
+  const enterpriseOfferWithoutBookingsLimit = applicableEnterpriseOffers.find(offer => !offerHasBookingsLimit(offer));
+  if (enterpriseOfferWithoutBookingsLimit) {
+    return enterpriseOfferWithoutBookingsLimit;
+  }
+
+  // use offer that has largest remaining balance for user
+  const enterpriseOfferWithUserBookingsLimit = applicableEnterpriseOffers
+    .filter(offer => offer.remainingBalanceForUser)
+    .sort((a, b) => a.remainingBalanceForUser - b.remainingBalanceForUser)
```

Comment:
nit: You could just do `b.remainingBalanceForUser - a.remainingBalanceForUser)[0]` so to avoid the additional `.reverse`

Reply:
Good suggestion :) Addressed in local commit and will push up before merge.